tasks/extract_filepaths_for_granule/schemas/input.json

```diff
@@ -0,0 +1,68 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "The input for the extract_filepaths_for_granule Lambda.",
+    "type": "object",
+    "properties": {
+      "granules": {
+        "description": "A list of objects representing individual files.",
+        "type": "array",
+        "items": [
+          {
+            "type": "object",
+            "properties": {
+              "granuleId": {
+                "description": "The ID of the granule.",
+                "type": "string"
+              },
+              "version": {
+                "description": "The version of the granule.",
+                "type": "string"
+              },
+              "files": {
+                "description": "Description of the files within the given granule.",
+                "type": "array",
+                "items": [
+                  {
+                    "type": "object",
+                    "properties": {
+                      "name": {
+                        "description": "The name and extension of the file.",
+                        "type": "string"
+                      },
+                      "key": {
+                        "description": "The key of the file.",
```

Comment:
Maybe expand this to the `The S3 key of the file (full path and file name less the bucket name).`

Reply:
changed